plasmapy/formulary/collisions.py

```diff
@@ -2300,6 +2321,93 @@ def Lorentz_collision_frequency(self):
             / (self.test_particle.mass**2 * self.v_a**3)
         ).to(u.Hz)
 
+    @cached_property
+    def v_e(self):
+        r"""Average momentum relaxation rate for a slowly flowing Maxwellian
+        distribution of electrons.
+
+        In order to use this attribute, ``test_particle`` must be an electron
+        and ``field_particle`` must be an ion.
+
+        :cite:t:`braginskii:1965` provides a derivation of this as an
```

Comment:
I'd cite Callen here - Braginskii doesn't really derive this I don't think (I can send you that paper if you really feel like diving into theoretical plasma physics from the 1960's...)